koku/masu/processor/parquet/parquet_report_processor.py

```diff
@@ -149,6 +143,14 @@ def manifest_id(self):
         """The manifest id."""
         return self._manifest_id
 
+    @property
+    def report_status(self):
+        if self.manifest_id:
+            return CostUsageReportStatus.objects.get(
+                report_name=Path(self._report_file).name, manifest_id=self.manifest_id
+            )
```

Comment:
Could this value be cached on the instance? Assuming `self._report_name = None` is added to the `__init__`:
```suggestion
    def report_status(self):
        if self.manifest_id:
            if self._report_name is None:
                self._report_name = CostUsageReportStatus.objects.get(
                    report_name=Path(self._report_file).name, manifest_id=self.manifest_id
                )
        return self._report_name
```

Reply:
I switched this to a cached_property instead.